Add description of vshard.enable/disable (#3019)
tarantool/doc: doc/reference/reference_rock/vshard/vshard_router.rst

            status: available_rw
        ...

..  _router_api-enable:

..  function:: vshard.router.enable()

    Manually allow access to the router API, revert
    :ref:`vshard.router.disable() <router_api-disable>`.

>         .. NOTE::
> 
>            ``vshard.router.enable()`` cannot be used for enabling the router
>            API which was automatically disabled due to running configuration
>            process.

Reviewer: ```suggestion
        ..  note::

            ``vshard.router.enable()`` cannot be used for enabling a router
            API that was automatically disabled due to a running configuration
            process.
```
1. We use `..  note::` now: https://www.tarantool.io/en/doc/latest/contributing/docs/markup/admonitions/
2. A running configuration process is `router.cfg()`, right? Let's mention this.

..  _router_api-disable:

..  function:: vshard.router.disable()

>     Manually restrict access to the router API. Disabling makes all
>     its methods, excluding :ref:`vshard.router.cfg() <router_api-cfg>`,
>     :ref:`vshard.router.new() <router_api-new>`,
>     :ref:`vshard.router.enable() <router_api-enable>` and
>     :ref:`vshard.router.disable() <router_api-disable>`, to throw a Lua
>     error in case of invoking them. The error object has a name attribute
>     equal to ``ROUTER_IS_DISABLED``.

Reviewer: ```suggestion
    Manually restrict access to the router API. When the API is disabled,
    all its methods throw a Lua error,
    except :ref:`vshard.router.cfg() <router_api-cfg>`,
    :ref:`vshard.router.new() <router_api-new>`,
    :ref:`vshard.router.enable() <router_api-enable>` and
    :ref:`vshard.router.disable() <router_api-disable>`. The error object's name attribute is
    ``ROUTER_IS_DISABLED``.
```
Russian is a wordy language, and English is not. Please try to [use fewer words and simpler constructions](https://www.tarantool.io/en/doc/latest/contributing/docs/style/#concise-is-good) whenever possible.

    The router is enabled by default. However, it is automatically and

>     forcefully disabled until configuration is finished as accessing the

Reviewer: ```suggestion
    forcefully disabled until the configuration is finished, as accessing the
```
